linux/shim/service.go

```diff
@@ -46,7 +46,10 @@ func NewService(path, namespace, address string) (*Service, error) {
 		client = events.NewEventsClient(conn)
 	} else {
 		client = &localEventsClient{
-			emitter: evt.GetPoster(context),
+			// TODO(stevvooe): This doesn't look entirely correct. Where do
+			// these events go? Looks like a bug, as we were pulling nil poster
+			// out of context.
+			forwarder: evt.NewExchange(),
```

Comment:
It should be provided by the `WithLocal(` option (which is not used anywhere atm).

Since it's already broken, the TODO will do for now.